data/formats-data.js

```diff
@@ -4779,8 +4779,8 @@ exports.BattleFormatsData = {
 	},
 	keldeoresolute: {},
 	meloetta: {
-		randomBattleMoves: ["relicsong","closecombat","knockoff","uturn","calmmind","psyshock","hypervoice","shadowball"],
-		randomDoubleBattleMoves: ["relicsong","closecombat","calmmind","psyshock","thunderbolt","hypervoice","uturn","protect"],
+		randomBattleMoves: ["uturn","calmmind","psychic","psyshock","hypervoice","shadowball","focusblast"],
```

Comment:
Meloetta has higher usage on Psychic than Psyshock, both in its tier and in doubles, but OK.